wow/sql/alerts_district_litigation.sql

```diff
@@ -0,0 +1,26 @@
+SELECT 
+	bbl,
+	x.housenumber,
+	x.streetname,
+	x.boro,
+	l.caseopendate,
+	l.casetype,
+	l.respondent,
+	count(*) over () as total_litigations
+FROM wow_bldgs as x
+LEFT JOIN pluto_latest_districts AS pld USING(bbl)
```

Comment:
here I think we can also use `wow_indicators` instead of wow_bldgs and pluto_latest_districts